app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -23,8 +23,7 @@ def call
       private
 
       def create_authentication_parameters
-        @logger.debug(LogMessages::Authentication::AuthnJwt::CREATING_AUTHENTICATION_PARAMETERS_OBJECT.new)
-        @authentication_parameters ||= @jwt_configuration.create_authentication_parameters(@authenticator_input)
+        @authentication_parameters ||= @jwt_configuration.authentication_parameters(@authenticator_input)
       end
```

Comment:
please change this to:
```
def create_authentication_parameters
  authentication_parameters
end

def authentication_parameters
  @authentication_parameters ||= @jwt_configuration.authentication_parameters(@authenticator_input)
end
```

this first method is used to tell the story in the `call` method. the second method is used for memoizing the `authentication_parameters` object. when you have a memoizing method it needs to match the name of the memoized object